src/GraphQL/Execution/SubscriptionExecutionStrategy.cs

```diff
@@ -166,7 +166,7 @@ await listener.AfterExecutionAsync(context)
             IEnumerable<object> path,
             Exception ex = null)
         {
-            var error = new ExecutionError(message, ex);
+            var error = new UnhandledError(message, ex);
```

Comment:
Unhandled exceptions within `SubscriptionExecutionStrategy` should throw an `UnhandledException`